README.md

```diff
@@ -4,12 +4,12 @@ An IntelliJ-based IDEs plugin for [**Bloblang**](https://www.benthos.dev/docs/gu
 **Bloblang** is a  mapping language developed by @jeffail as part of [**Benthos**](https://github.com/benthosdev/benthos), a high performance and resilient stream processor.
 
 ## Features
-This pliugin recognizes the files `*.blobl` as bloblang files and check and highlight the syntax of these files. Fot more details tou can check the [CHANGELOG](CHANGELOG.md).
+This plugin recognizes the files `*.blobl` as bloblang files checking the highlight and syntax of these files. Fot more details you can check the [CHANGELOG](CHANGELOG.md).
 
 ## Install
 ### From Marketplace
-This alternative the easier to keep the plugin updated. Since the plugin is in the very beginning of its development, it isn't in the stable channel but in the EAP. For installing plugins from this channel you have to add the repository to your plugins repositories list.
-1. Press `Ctrl+Alt+S` (or `⌘ ,` in Mac) to open the IDE settings and select **Plugins**.
+This alternative keeps the plugin updated. Since the plugin is in the beginning of its development, it isn't in the stable channel but in the EAP. For installing plugins from this channel you have to add the repository to your plugins repositories list.
```

Comment:
Keep a plugin updated is a manual action, the IDE will only notify about updates. The word very is correct.

Reply:
Fixed